app/components/FormInput/styles.js

```diff
@@ -103,42 +103,51 @@ export const InputWrapper = styled.div`
 export const InputStyled = styled(Input)`
   width: 100% !important;
   height: 36px !important;
-  border: 0 !important;
   padding: 8px !important;
   outline: 0 none !important;
   transition: border-color .2s linear !important;
-  border-bottom: 1px solid #000 !important;
+  border: 1px solid #000 !important;
   -webkit-font-smoothing: antialiased !important;
-  border-radius: 0 !important;
+  border-radius: 3px !important;
   background: transparent !important;
+  color: #000 !important;
   &:focus {
     box-shadow: none !important;
+    border-radius: 3px !important;
     border: 1px solid #000 !important;
-    border-radius: 2px !important;
   }
   &:hover {
     transition: all 300ms ease;
     background-color: #ddd;
   }
+  ${({ login }) => login && css`
```

Comment:
We better use extends for this case
```suggestion
LoginInputStyled = styled(InputStyled)`
...
`;
```